lib/generic/states.rb

```diff
@@ -211,9 +240,14 @@ class Encounter < State
 
         metadata 'codes', type: 'Components::Code', min: 1, max: Float::INFINITY
 
+        def initialize(context, name)
+          super(context, name)
```

Comment:
a constructor that just calls super with the same args doesn't need to be explicitly defined